docs/production-deployment/cloud/operation-api.mdx

```diff
@@ -105,12 +101,27 @@ Use [gRPC](https://grpc.io/docs/) to compile and generate code in your preferred
    - Locate the Python files (.py) generated in your project directory.
    - Import these files into your Python application where you intend to interact with the Temporal Cloud API.
 
-2. **Use the API:**
+2. Use the API:
    - Use the classes and methods defined in the imported files to communicate with the Temporal Cloud services.
    - Ensure to handle any required authentication or configuration as needed for Temporal Cloud.
 
 This approach can be adapted for other programming languages by following their respective import and usage conventions for the generated code files.
 
+### Using HTTP
+
+[The HTTP API](https://rest.saas-api.tmprl.cloud/docs/httpapi.html#description/introduction) supports the same operations as the existing [gRPC API](#using-grpc), but it's usable via standard HTTP methods and authentication. This may be a more convenient option if you are writing automation scripts for CI/CD or you can't use gPRC due to network policies, proxies, tooling gaps, or language/runtime constraints. Since it's standard HTTP, it's language agnostic giving you the ability to run cloud operations consistently.
```

Comment:
```suggestion
[The HTTP API](https://rest.saas-api.tmprl.cloud/docs/httpapi.html#description/introduction) supports the same operations as the [gRPC API](#using-grpc), but it's usable via standard HTTP methods and authentication. This may be a more convenient option if you are writing automation scripts for CI/CD or you can't use gPRC due to network policies, proxies, tooling gaps, or language/runtime constraints. Since it's standard HTTP, it's language agnostic, giving you the ability to run cloud operations consistently.
```